ansible/roles/ci/deploy/upload_debs/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Defaults aptly url
+
+aptly_url: http://10.10.10.62:8080
+repository_name: shadow-private
+gpg_password: shadow
```

Comment:
for now it is OK. But if it is possible to keep password out of public repository it will be a plus using "more secure options"